src/screens/DataApi/DataCoverageTable.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+
+import CheckIcon from '@material-ui/icons/Check';
+
+import {
+  TextWithTooltipContainer,
+  UnderlinedText,
+  StyledTableHead,
+  MetricTableCell,
+  AvailabilityTableCell,
+  MetricTableRow,
+  MetricTableCellHeader,
+} from './DataCoverageTable.style';
+import coverageSummary from './coverage-summary.json';
+import { formatPercent } from 'common/utils';
+import { InfoTooltip } from 'components/InfoTooltip';
+import { pluralize } from 'components/Explore/utils';
+import { RegionType } from 'common/regions';
+
+export const COVERAGE_SUMMARY = coverageSummary as MetricCoverage[];
+
+interface MetricField {
+  displayName: string;
+  field: string;
+}
+
+export const FIELDS: MetricField[] = [
+  { displayName: 'Overall risk level', field: 'riskLevels.overall' },
+  { displayName: 'Daily new cases', field: 'metrics.caseDensity' },
+  { displayName: 'Infection rate', field: 'metrics.infectionRate' },
+  { displayName: 'Positive test rate', field: 'metrics.testPositivityRatio' },
+  { displayName: '% Vaccinated', field: 'metrics.vaccinationsInitiatedRatio' },
+  { displayName: 'ICU capacity used', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Hospitalization usage', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Deaths', field: 'actuals.deaths' },
+  { displayName: 'Total testing volume', field: 'actuals.positiveTests' },
+];
+
+export const FIELDS_PRETTY_NAMES = FIELDS.map(field => {
+  const record = COVERAGE_SUMMARY.find(record => field.field === record.name);
```

Comment:
nit: consider filtering the fields that have records and then mapping to make the sequence easier to read

```ts
// I'm not sure if creating the functions below is worth the trouble...
FIELDS
  .filter(hasRecord)
  .map(fieldToRecord)
  .filter((record): record is MetricCoverage);


```